.ci.sh

```diff
@@ -15,14 +15,11 @@ source python/etc/profile.d/conda.sh  # to be able to call conda
 conda activate venv_sct  # reactivate conda for the pip install below
 
 echo *** UNIT TESTS ***
-pytest
-
-echo *** INTEGRATION TESTS ***
 pip install coverage
 echo -ne "import coverage\ncov = coverage.process_startup()\n" > sitecustomize.py
 echo -ne "[run]\nconcurrency = multiprocessing\nparallel = True\n" > .coveragerc
 COVERAGE_PROCESS_START="$PWD/.coveragerc" COVERAGE_FILE="$PWD/.coverage" \
```

Comment:
TODO review this.. smells fishy..

Reply:
Maybe related to #2702. I am questioning the relevance of those `echo` lines and also seems like the coverage is not updated at all. 

Out of the scope of this PR though.